debian/kano-connect.postinst

```diff
@@ -26,13 +29,17 @@ case "$1" in
         mv $TMP_FILE /etc/sudoers.d/
 
         if [ -e /etc/kanux_version ]; then
-
             # setting wireless module parameters
             cp /usr/share/kano-network/etc/modprobe.d/* /etc/modprobe.d/
 
-            # updating the /etc/network/interfaces
-            cp /usr/share/kano-network/etc/network/* /etc/network/
+            # remove previous interfaces file, for udhcpd5 migration
+            if [ -f $interfaces_file ]; then
+                rm -f $interfaces_file
+            fi
 
+            if [ -f $interfaces_file_backup ]; then
+                rm -f $interfaces_file_backup
```

Comment:
Do these files have anything to do with the way that we set the parental settings. I think not because I think that we actually care about the `/etc/resolvconf/interface-order` but I'm just a little concerned because I don't understand why we have an `interfaces` and an `interfaces-backup` to start with.

Reply:
The file `interfaces` originally comes from a postinst script in the`ifupdown` Debian package, with an empty stub. For historic reasons, we saved a backup copy of it to be safe, but we never used the backup version. We did for the original file to force the connection hooks, which seemed to be the cause for troubles during a shutdown. It would still be useful to use for special cases like fixed IP addresses.

I have documented it briefly [here](https://github.com/KanoComputing/kano-toolset/pull/254/commits/53a67cd5a5c83c91153f98a3b4ce0e90c40bab8e)